eras/alonzo/impl/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 1.10.2.1
```

Comment:
This is a visible change, so minor version needs to be bumped (in cabal file too)

Although depending on my comment about the actual instance, we might be able to keep the patch bump and remove the instance instead.
```suggestion
## 1.10.3.0
```